web/src/modules/deployments/index.ts

```diff
@@ -172,6 +174,19 @@ export const approveStage = createAsyncThunk<
   await thunkAPI.dispatch(fetchCommand(commandId));
 });
 
+export const skipStage = createAsyncThunk<
+  void,
+  { deploymentId: string; stageId: string }
+>("deployments/skip", async (props, thunkAPI) => {
+  const { commandId } = await deploymentsApi.skipStage(props);
+  await thunkAPI.dispatch(fetchCommand(commandId));
+});
+
+export const updateSkippableState = createAsyncThunk<void, { stageId: string }>(
+  "deployments/skippable",
+  () => {}
+);
```

Comment:
nits, `updateSkippableState` is a bit odd and hard to understand since the name is too general. How about making this action accepts `skippable: bool` besides `stageId`? Or rename this to `lockSkippableState`? 🤔